### PR TITLE
More verbose switch sync logging

### PR DESCRIPTION
**Pull request by surabujin**
Add __str__ method for object's produced by message_utils.py. Also add
test for switch sync request.

### CODE REVIEW

File: services/topology-engine/queue-engine/topologylistener/message_utils.py [hunk 1]
```diff
@@ -37,8 +37,15 @@
 
 
 class Abstract(model.JsonSerializable):
-    def to_json(self):
-        return json.dumps(self, cls=model.JSONEncoder)
+    def __str__(self):
+        return '<{}: {}>'.format(type(self).__name__, self.to_json(
+            uniformly=True))
+
+    def to_json(self, uniformly=False):
```
Comment (timofei-durakov) [inline services/topology-engine/queue-engine/topologylistener/message_utils.py]: why not to name this flag as sort_keys and pass it as is to json.dumps?

File: services/topology-engine/queue-engine/topologylistener/messageclasses.py [hunk 2]
```diff
@@ -238,7 +238,7 @@ def handle_link_props(self):
                 raise exc.NotImplementedError(
                     'link props request {}'.format(self.get_message_type()))
         except CypherSyntaxError as e:
-            logger.exception('Invalid request: ', e)
+            logger.exception('Request produce invalid cypher query')
```
Comment (timofei-durakov) [inline services/topology-engine/queue-engine/topologylistener/messageclasses.py]: do we hide Cypher exception here and do not have traceback?

Comment (nikitamarchenko) [reply services/topology-engine/queue-engine/topologylistener/messageclasses.py]: according to docs https://docs.python.org/2/library/logging.html#logging.Logger.exception we will have all exception data and traceback

File: services/topology-engine/queue-engine/topologylistener/message_utils.py (hunk 1 above)
Comment (surabujin) [reply services/topology-engine/queue-engine/topologylistener/message_utils.py]: Good question. :) Will change it in proposed way.

**Pull request merged**